src/Pages/danbipark/Login/LoginP.scss

```
@@ -0,0 +1,140 @@
+@import url(https://fonts.googleapis.com/css2?family=Lobster&display=swap);
+@import "../../../Styles/Common.scss";
+@import "../../../Styles/Reset.scss";
+
+* {
+  box-sizing: border-box;
+}
+
+$color-lightgray: #fafafa;
+$color-midgray: #a5a3a3;
+$color-darkgray: #3a3a3a;
+$color-border: #e0e0e0;
+$color-text-add: #8e8e8e;
+$color-hashtag: #00376b;
+$color-action: #0095f6;
+
+$avtar-size: 36px;
+$avatar-my-size: 66px;
+$avatar-friends: 32px;
+$avatar-padding: 16px;
+
+$font-size: 14px;
+$font-padding: 4px;
+
+body {
+  background-color: $color-lightgray;
+  margin: 0;
+}
+
+.Login {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  background-color: $color-lightgray;
```

css속성 순서를 수정해주면 더 좋을 것 같습니다!Layout Properties (position, float, clear, display)

```suggestion
  height: 100vh;
  display: flex;
  justify-content: center;
  align-items: center;
```

---

앗 Layout 프로퍼티가 Box Model 프로퍼티보다 상위라고 알고있는데 아닌가요?~